app/src/main/res/values-land/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<item name="hero_aspect_ratio" type="string">H,32:9</item>
```

Comment:
I think this should be just `<string name="hero_aspect_ratio">H,32:9<string>`. But I think I like keeping it in the `dimens.xml` file - even though it's a string, it is defining a dimension. (Same with the non-landscape version of course).

Reply:
Fair enough.